### DISCUSSION TITLE
Can 'Focus sentinel' links in Tooltip be removed/hidden from screen reader?

### DISCUSSION THREAD

**Discussion by BenjaminWFox-Lumedic** [carbon-design-system/carbon#8367] [Help] [answered]
This is specifically related to the [React `Tooltip` component](https://react.carbondesignsystem.com/?path=/story/components-tooltip--default-bottom).

When implemented, the popup content is flanked by two "Focus sentinel" `span` elements (see markup below). I'm not clear on the exact role of these (something to do with focus trapping?) but they create a confusing experience when navigating content via keyboard/screen reader since both tabbing in and out of the popup content announces "Link Focus Sentinel". 

```html
<div>
  <span tabindex="0" role="link" class="bx--visually-hidden">Focus sentinel</span>
  <div class="bx--tooltip ..." id="..." data-floating-menu-direction="bottom" role="tooltip" ...>...</div>
  <span tabindex="0" role="link" class="bx--visually-hidden">Focus sentinel</span>
</div>
```

Questions:
- Is there a way to disable these elements from showing up?
- Should these elements be `aria-hidden`?

**Comment by jnm2377** [comment, marked as answer]
Hey @BenjaminWFox-Lumedic can you provide some more info on what version of Carbon you're using and also what browser & screen reader you're using? When I test Voice over on Chrome, it doesn't announce "Link Focus Sentinel".


https://user-images.githubusercontent.com/32556167/129826330-393be19a-e86b-4098-840b-259753b9825e.mov

**Reply by BenjaminWFox-Lumedic** [reply]
Thanks for following up @jnm2377. I don't see this behavior happening anymore in our own storybook (OSX Voiceover, latest Chrome) and the project where it was originally an issue was scrapped so I can't run that specific code anymore.

It may also be we had an older version of Carbon we were using at the time. I'll create something new with better detail if I run in to the issue in the future!
